File: devcamp2018/app/src/main/java/com/psato/devfest/data/entity/Show.java
package com.psato.devfest.data.entity;

import com.google.gson.annotations.SerializedName;

/**
 * Created by psato on 29/10/16.
 */

public class Show {

    @SerializedName("title")
    private String mTitle;
    @SerializedName("year")
    private Integer mYear;
    @SerializedName("ids")
    private ShowIds mIds;


    public String getTitle() {
        return mTitle;
    }

    public Integer getYear() {
        return mYear;
    }

    public ShowIds getIds() {
        return mIds;
    }
}

File: devcamp2018/app/src/main/java/com/psato/devcamp/data/entity/Show.kt
package com.psato.devcamp.data.entity

import com.google.gson.annotations.SerializedName

/**
 * Created by psato on 29/10/16.
 */

class Show {

    @SerializedName("title")
    var title: String? = null
    @SerializedName("year")
    var year: Int? = null
    @SerializedName("ids")
    var ids: ShowIds? = null
}
